src/main/java/io/jenkins/plugins/services/impl/HttpClientJiraIssues.java

```diff
@@ -44,21 +45,34 @@ public String getHttpContent(String url, List<Header> headers) {
     return super.getHttpContent(url, headers);
   }
 
+  @Override
+  protected boolean isValidStatusCode(int statusCode) {
+    return statusCode == HttpStatus.SC_OK || statusCode == HttpStatus.SC_BAD_REQUEST;
+  }
+
   public JiraIssues getIssues(String pluginName) throws IOException {
     return getIssues(pluginName, 0);
   }
 
   public JiraIssues getIssues(String pluginName, int startAt) throws IOException {
     int maxResults = 100;
+    String component = pluginName.replaceAll("-plugin$", "") + "-plugin";
```

Comment:
Does this actually need stripping? We should be getting the artifact at this point and I assume jira will match that so we might be breaking a few plugins by stripping this?

Reply:
I... I don't know, someone in the past warned me about some plugins having -plugins, and this seemed like the easy fix.

Reply:
ah right I see the issue, otherwise we get double plugin, it's fine should work right